### ISSUE TITLE
JetEngine: Custom Post Type Meta Field (Select)

### ISSUE THREAD

**Issue by iamdanbarrett** [CrocoBlock/suggestions#3840]
There needs to be the ability to add items in the select options that are non-selectable to break up large lists such as countries.

Example:

**Europe (non selectable)**
England
France
Italy
**Africa (non selectable)**
Kenya
Morrocco
Nigeria

And so on....

**Comment by nreljed** [comment]
It is a problem of jetengine scalability which will grow endlessly if nobody does anything, they will eternally make endless modifications.
unless they adopt... the only easy solution is fast and scalable
the Select2 open source library

[Select2 ](https://github.com/select2/select2) 

look at the answer to your questions are there [Preview dropdown](https://select2.org/dropdown)

ACF does not worry too much about select types and [has adopted select2](https://www.advancedcustomfields.com/resources/select/)

it is a problem of deciding whether or not to adopt select2 and why has it not been done yet?

Sometimes you have to have all the same tools as the others competitors to succeed.

for example try to make a jetengine select with all the countries ..

without integrated select 2 search bar it's over! jetengine has reached its limits.

select2 even allows you to do exotic things like add icons (country flag)

basically if a dev passes here  give us reassuring messages :)

![Plan de travail – 4](https://user-images.githubusercontent.com/82481592/129753673-c9e7737d-9611-4db6-bed7-fa6d83380ae1.jpg)

#3885

**Comment by kochhase** [comment]
+1

**Comment by P258tiwari** [comment]
+1

**Comment by sara-kariv** [comment]
+1
